test/jdk/tools/jpackage/share/ServiceTest.java

```diff
@@ -248,6 +262,9 @@ public void testAddLFromAppImage(boolean mainLauncherAsService) {
                     if (mainLauncherAsService) {
                         cmd.addArgument("--launcher-as-service");
                     }
+                    if (isMacAppStore) {
+                        cmd.addArgument("--mac-app-store");
+                    }
```

Comment:
Do we still need this change?

Reply:
Looks like in the case of predefined app image packaging, when "--mac-app-store" is used with the predefined app image but not used in the terminal packaging command, jpackage still adds service files to the package. Looks like a bug.

We added "mac-app-store" property to the app image file in [JDK-8286850](https://bugs.openjdk.org/browse/JDK-8286850). Do we still need it? Why do we want to know that the predefined app image was created with "--mac-app-store" option? In app image bundling, this option affects signing identities only, right?

Reply:
It is a bug in verification code. Test thinks that we should have 2 top installation directory, but we got 1. This is expected since predefined app image was generated with "--mac-app-store".

Reply:
Yes, we still need it. Yes it affects signing. It is used so we can sign predefined app image correctly for Mac App Store if it was generated with "--mac-app-store".